packages/autocomplete/package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "pcln-autocomplete",
-  "version": "2.0.5",
+  "version": "2.1.0",
   "description": "React autocomplete component built with Downshift",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
   "scripts": {
-    "prepare": "rollup -c",
-    "clean": "rm -rf node_modules dist",
+    "prepare": "npm run build",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:es",
```

Comment:
Can we use [npm-run-all](https://www.npmjs.com/package/npm-run-all)? Then it'd just be `npm-run-all build:*` and parallelized.

Reply:
TIL. This is really cool